README.md

```diff
@@ -112,7 +112,7 @@ Various inputs are defined in [`action.yml`](action.yml) to let you configure th
 | Name | Description | Default |
 | - | - | - |
 | `repo-token` | Token to use to authorize label changes. Typically the GITHUB_TOKEN secret, with `contents:read` and `pull-requests:write` access | `github.token` |
-| `configuration-path` | The path to the label configuration file | `.github/labeler.yml` |
+| `configuration-path` | The path to the label configuration file. If not found in the current working directory, will read from the source repository via the Github API. | `.github/labeler.yml` |
```

Comment:
By default action searches for configuration in .github folder. We should better mention this instead of `working directory`.

Reply:
`working directory` would be correct, though? If `configuration-path` is not specified, the chart will read from `.github/labeler.yml`, though, and that's already documented in the README.

Reply:
@lrstanley `working directory` might be misleading as it stands for the root directory of the project. 
`If file is not found at the provided path to config file, will read from the source repository via the GitHub API` - is what we came up with.